game_board/database/test_db.py

```diff
@@ -0,0 +1,163 @@
+"""
+python manage.py test game_board.database.tests_db
+"""
+
+from game_board.database import game_board_db as mongo
+from django.test import TestCase
+
+class BColors:
+    # Colors for printing
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+board = {
+    "game_id": "60afce36-085a-11eb-b6ab-acde48001122",
+    "graph": {
+        "nodes": "node4(node2(node3)(node1))(node6(node5))",
+        "node_points": {
+            "node1": 1,
+            "node2": 2,
+            "node3": 3,
+            "node4": 4,
+            "node5": 5,
+            "node6": 6
+        },
+        "gold_node": "node5",
+        "balanced": True
+    },
+    "player_ids": [
+        "id2",
+        "id3",
+        "id4",
+        "id5"
+    ],
+    "player_names": [
+        "naomi",
+        "kulsoom",
+        "nick",
+        "ryan"
+    ],
+    "player_points": {
+        "id2": 2,
+        "id3": 2,
+        "id4": 3,
+        "id5": 10
+    },
+    "turn": "id2",
+    "cards": {
+        "id2": [
+            "card1",
+            "card2",
+            "card3"
+        ],
+        "id3": [
+            "card1",
+            "card2",
+            "card3"
+        ],
+        "id4": [
+            "card1",
+            "card2",
+            "card3"
+        ],
+        "id5": [
+            "card1",
+            "card2",
+            "card3"
+        ]
+    },
+    "gold_node": False,
+    "difficulty": "Medium",
+    "num_players": 4,
+    "online": True,
+    "curr_data_structure": "AVL",
+    "selected_data_structures": [
+        "AVL",
+        "Stack"
+    ],
+    "timed_game": False,
+    "seconds_until_next_ds": 60,
+    "time_created": "07/10/2020 00:05:47",
+    "end_game": False
+}
+
+FAIL_PHRASE = 'nah bro idk about it'
```

Comment:
So are we keeping this into master lol?

Reply:
I would need to change the API code if this is changed. So please keep me in the loop.

Reply:
It's functional, but its fine if you want to change it. It was requested by Maksim to have fun in the coding, and I didn't think it was an issue to implement. It sounds like a significant enough change at this point if it involves the tests, calls, and the stuff in the API, so maybe we can create an issue for it in this upcoming sprint?